### PR TITLE
tests: include details to a set of tests - part 8

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Adding details to spread tests.

### CODE REVIEW

File: tests/main/upgrade-from-2.15/task.yaml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 summary: Ensure upgrades from snapd 2.15 work
 
+details: |
+    Check that upgrading core from a very old verios (2.15) to the
+    current one works. Verify that snap confine profiles are fine
+    after upgrade and ensure that the old/obsolete snap-confine
+    appamor profile got removed
```
Comment (MiguelPires) [inline tests/main/upgrade-from-2.15/task.yaml:7]:
```suggestion
    Check that upgrading core from a very old version (2.15) to the
    current one works. Verify that snap confine profiles are fine
    after upgrade and ensure that the old/obsolete snap-confine
    apparmor profile was removed
```

**Review by MiguelPires — APPROVED** [review]
Thank you, just some minor typo fixes and rewording suggestions

File: tests/main/snap-seccomp-blocks-tty-injection/task.yaml [hunk 2]
```diff
@@ -1,5 +1,9 @@
 summary: Ensure that the snap-seccomp blocks tty command injection
 
+details: |
+    Check it is not allowed to make tty command injection. Verify snap-seccomp
+    generated "Operation not permitted" message when it denies the injection.
```
Comment (bboozzoo) [inline tests/main/snap-seccomp-blocks-tty-injection/task.yaml:5]:
```suggestion
    generated a profile which results in  "Operation not permitted" message when 
    attempt to inject a tty command is denied.
```

**Review by bboozzoo — APPROVED** [review]
LGTM

**Pull request merged**